VERSION

```diff
@@ -1,3 +1,3 @@
 # The release version and build number of the stackdriver-agent package.
 PKG_VERSION=5.5.2
-PKG_BUILD=1002
+PKG_BUILD=1003
```

Comment:
Please make sure the new PR title becomes the resulting squash-and-merge commit title.